packages/console/src/components/CopyToClipboard/index.module.scss

```diff
@@ -42,6 +42,18 @@
       .copyToolTipAnchor {
         margin-left: _.unit(3);
       }
+
+      .iconButton {
+        height: 24px;
+        width: 24px;
+
+        .icon {
+          svg {
+            width: 16px;
+            height: 16px;
+          }
```

Comment:
Can this be omitted, as the SVG has a size of 16x16 by default

Reply:
No, the svg under `IconButton` will be set to 20x20 by default.